samples/Plugin.Maui.Audio.Sample/ViewModels/AudioRecorderPageViewModel.cs

```diff
@@ -52,6 +55,84 @@ public bool IsRecording
 		this.dispatcher = dispatcher;
 	}
 
+	ChannelTypesViewModel selectedChannelType;
+	public ChannelTypesViewModel SelectedChannelType
+	{
+		get => selectedChannelType;
+		set
+		{
+			selectedChannelType = value;
+			NotifyPropertyChanged();
+		}
+	}
+	public ObservableCollection<ChannelTypesViewModel> ChannelTypes { get; set; } = new ObservableCollection<ChannelTypesViewModel>(Enum.GetValues(typeof(ChannelTypes)).Cast<ChannelTypes>().Select(x => new ChannelTypesViewModel()
```

Comment:
It's fairly minor but I don't think these need to be ObservableCollection? The contents never change right? If not then it can just be a List